validator/src/main.rs

```diff
@@ -1046,6 +1053,7 @@ pub fn main() {
             (
                 SocketAddr::new(rpc_bind_address, rpc_port),
                 SocketAddr::new(rpc_bind_address, rpc_port + 1),
+                // +2 is skipped to avoid a conflict with the websocket port (which is +2) in web3.js
```

Comment:
```suggestion
                // +2 is skipped to avoid a conflict with the secure websocket port (which is +2) in web3.js
```

Reply:
(This is done with: https://discord.com/channels/428295358100013066/439194979856809985/755473343163138208)

Nice catch. I did so wss => ws intentionally.

Well, I'm not quite sure. Does `solana-validator` natively handle TLS-encpuslated websocket (wss) connections? I thought we're listening with a different port from the normal RPC listening address because we can't handle websockets in general with the single shared port, not specifically only secure websockets.

Could you enlighten me a bit?

Reply:
Hmm, actually not sure if this is still needed: https://github.com/solana-labs/solana-web3.js/commit/5b4aa21388fae4ad16c9c883ede1a701d7575b1e
@mvines can you clarify?

Reply:
I think https://github.com/solana-labs/solana/issues/12250 sums up the problem.  We can make this go away any time, just a bit of server ops config file work 🔧